File: rtmp/src/main/java/net/ossrs/rtmp/ConnectCheckerRtmp.java
package net.ossrs.rtmp;

/**
 * Created by pedro on 25/01/17.
 */
public interface ConnectCheckerRtmp {

  void onConnectionStartedRtmp(String rtmpUrl);

  void onConnectionSuccessRtmp();

  void onConnectionFailedRtmp(String reason);

  void onNewBitrateRtmp(long bitrate);

  void onDisconnectRtmp();

  void onAuthErrorRtmp();

  void onAuthSuccessRtmp();
}

File: rtmp/src/main/java/com/pedro/rtmp/utils/ConnectCheckerRtmp.kt
package com.pedro.rtmp.utils

/**
 * Created by pedro on 8/04/21.
 */
interface ConnectCheckerRtmp {
  fun onConnectionStartedRtmp(rtmpUrl: String)
  fun onConnectionSuccessRtmp()
  fun onConnectionFailedRtmp(reason: String)
  fun onNewBitrateRtmp(bitrate: Long)
  fun onDisconnectRtmp()
  fun onAuthErrorRtmp()
  fun onAuthSuccessRtmp()
}